pkg/postgres/schema/process_listening_on_ports.go

```diff
@@ -0,0 +1,57 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
```

Comment:
Is this one still being generated?  the line in `gen.go` is commented out so I suspect it is not.

Reply:
`gen.go` for PLOP is still in place, only one for ProcessIndicator was removed. PLOP doesn't generate the store itself due to `--schema-only`, but the db schema is still automated and in sync.